File: backend/services/operations-service/src/main/java/com/lmspilot/operations/OperationsServiceApplication.java
package com.lmspilot.operations;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableScheduling
public class OperationsServiceApplication {
    public static void main(String[] args) { SpringApplication.run(OperationsServiceApplication.class, args); }
}

File: backend/services/operations-service/src/main/kotlin/com/lmspilot/operations/OperationsServiceApplication.kt
package com.lmspilot.operations
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling
@EnableScheduling
@SpringBootApplication(scanBasePackages=["com.lmspilot"])
class OperationsServiceApplication
fun main(args:Array<String>) { runApplication<OperationsServiceApplication>(*args) }
